projects/microchip/curiosity_pic32mzef/mplab/aws_demos/nbproject/.gitignore

```diff
@@ -1,7 +1,3 @@
 private/
 Makefile-genesis.properties
-Makefile-impl.mk
-Makefile-local-pic32mz_ef_curiosity.mk
-Makefile-pic32mz_ef_curiosity.mk
-Makefile-variables.mk
```

Comment:
I didn't realize the .gitignore for tests was just copied from this one. We might want to put the .mk files back unless these files aren't relevant anymore. Sorry for the confusion.

Reply:
No worries